<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommissionDialog</class>
 <widget class="QDialog" name="CommissionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>533</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Commission Edit</string>
  </property>
  <widget class="QPushButton" name="commResetButt">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commDelButt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="noSaleEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="limitBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Limits</string>
   </property>
   <widget class="QLineEdit" name="minEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_3_2_2_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_3_2_2_4_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabel1_3_2_2_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Quick Fill</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Code</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeRelLineEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="groupBox3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>210</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Calculate</string>
   </property>
   <widget class="QComboBox" name="calculationMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>By Transaction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>By Invoice</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="commSaveButt">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="colorComboBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0-69</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70-79</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80-89</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90-99</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="messageArea">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>260</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3_2_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>31</width>
     <height>260</height>
    </rect>
   </property>
   <property name="text">
    <string>F&lt;BR&gt;L&lt;BR&gt;A&lt;BR&gt;W</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Q3Table" name="codeTable">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>260</width>
     <height>260</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="resizePolicy">
    <enum>Q3ScrollView::Manual</enum>
   </property>
   <property name="numRows">
    <number>10</number>
   </property>
   <property name="numCols">
    <number>10</number>
   </property>
   <property name="selectionMode">
    <enum>Q3Table::NoSelection</enum>
   </property>
   <property name="focusStyle">
    <enum>Q3Table::FollowStyle</enum>
   </property>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>0</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>6</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>7</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>8</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>9</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="textLabel1_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>N/S Fee:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Commission Description</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="groupBox2_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Break</string>
   </property>
   <widget class="QComboBox" name="breakMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>By Weight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>By Head</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>By Price</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>By Total</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLineEdit" name="commissionDescriptionEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Q3ListBox" name="commissionListBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>370</width>
     <height>260</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="testCommissionButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>minEdit</tabstop>
  <tabstop>maxEdit</tabstop>
  <tabstop>codeRelLineEdit</tabstop>
  <tabstop>calculationMode</tabstop>
  <tabstop>breakMode</tabstop>
  <tabstop>colorComboBox</tabstop>
  <tabstop>commSaveButt</tabstop>
  <tabstop>commResetButt</tabstop>
  <tabstop>commDelButt</tabstop>
  <tabstop>codeTable</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/commissiondao.h</include>
  <include location="local">dao/animalcodedao.h</include>
  <include location="local">q3listview.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
